tests/rbd_mirror/rbd_mirror_utils.py

```diff
@@ -57,7 +57,7 @@ def exec_cmd(self, **kw):
             if kw.get("ceph_args", True):
                 cmd = cmd + self.ceph_args
 
-            out = node.exec_command(
+            out, err = node.exec_command(
```

Comment:
return values could be only one if long_running is passed.

Reply:
Done